docs/utils/data.rst

```diff
@@ -22,9 +22,9 @@ cache) the data will only be downloaded if it is not already present in the
 cache. The tools can be instructed to obtain a new copy of data
 that is in the cache but has been updated online.
 
-The ``astropy`` cache is stored in a centralized place (on Linux machines by
-default it is ``$HOME/.astropy/cache``; see :ref:`astropy_config` for
-more details).  You can check its location on your machine::
+The ``astropy`` cache is stored in a centralized place (see :ref:`astropy_config`
```

Comment:
Why should the `cache` documentation point to `config` documentation?

Reply:
Because cache location is *configurable* ? in any case, this is not from this PR.

Reply:
The cache location is not configurable using `astropy` configuration items.

Reply:
Again, not from this PR. I'm already creating potentially massive merge conflicts for myself to handle once we finally circle back to #17640, I would really like for this to stay on topic.

Reply:
On current `main` this documentation has to refer to the `config` documentation because that's where the `XDG_CACHE_HOME` environment variable is described. If this pull request creates a separate documentation page for the environment variables then there's no reason for the paragraph here to refer to `config` documentation anymore.

Reply:
fair enough